source/supervisord.conf

```diff
@@ -27,6 +27,18 @@ priority=2
 command=elasticsearch
 priority=1
 
+[program:nutch]
+command=nutch startserver
+priority=1
+
+[program:rabbitmq]
+command=rabbitmq-server
+priority=1
```

Comment:
should set these to autostart=false.  by default we don't want to turn on this functionality.